rwengine/src/data/GameTexts.cpp

```diff
@@ -2,8 +2,8 @@
 
 GameString GameStringUtil::fromString(const std::string& str) {
     GameString s;
-    for (std::string::size_type i = 0u; i < str.size(); ++i) {
-        s += str[i];
+    for (const char &i : str) {
+        s += i;
```

Comment:
Is it possible to use `std::accumulate`?

Reply:
yeah, but code becomes ugly(lambda)